Automattic/sensei
# Add course title and site logo blocks, update layout

Fixes  #4377

### Changes proposed in this Pull Request

* Create a block that renders the title of the current page's parent course
* Create a site logo block that renders the existing 'custom logo' for the site if it's set, linking to the course page
* Update course theme header layout

### Testing instructions

* Open course lesson
* See course title in header
* To set site logo: hit the theme's Customize option, and upload a Site logo under Site Identity. (Note some themes might disable this)


### Screenshot / Video

<img width="1265" alt="image" src="https://user-images.githubusercontent.com/176949/143479176-3d8456fe-c7eb-4e2b-9df0-6cfd0a05b64c.png">
